prow/jobs.md

```diff
@@ -73,6 +73,35 @@ that matches `trigger` will suffice. This is useful if you want to make one
 command that reruns all jobs. If unspecified, the default configuration makes
 `/test <job-name>` trigger the job.
 
+## Presets
+
+[`Presets`] can be used to define commonly reused values for a subset of fields
+for PodSpecs and BuildSpecs. The subset of fields chosen was inspired by
+[PodPresets] which at time of writing are still in alpha. A preset config looks
+like:
+
+```yaml
+presets:
+- labels:                  # a job with these labels/values will have the preset applied
+    preset-foo-bar: "true" #   key:value pair must be unique among presets
+                           #   our convention is preset-name: "true"
+- env:                     # list of valid Kuberentes EnvVars
```

Comment:
`env`, `volumes`, and `volumeMounts` should not be preceded by `-` since all those section should be part of the same list element (starting at `- labels:`).

Reply:
🤦‍♂ good catch

Reply:
Done.  I also dropped the "our convention" comment so the `env:` key is closer to `labels:`, making it more obvious at a glance they're part of the same element